OWNERS_ALIASES

```diff
@@ -299,6 +288,8 @@ aliases:
   # sig-apps-emeritus:
   # - tnozicka
 
+  sig-autoscaling-maintainers:
+    - mwielgus
```

Comment:
isn't this overwritten by the immediately following stanza?

Reply:
whoops fixed.